.github/workflows/maven.yml

```diff
@@ -26,6 +26,7 @@ jobs:
     with:
       ff-goal: '-P run-its install'         # site use project version for reporting
       verify-goal: '-P run-its install'     # should be the same as for first build
+      maven-matrix: '[ "3.5.4", "3.9.1" ]'  # Maven versions matrix for verify builds
```

Comment:
build will take about 1 hour 😄

Reply:
yes, but at last I would like to test somehow once